ssc/cmod_pvsamv1.cpp

```diff
@@ -2238,6 +2250,16 @@ void cm_pvsamv1::exec()
                 }
 			}
 
+            // Re-compute PV AC forecast for AC connected batteries
```

Comment:
@sjanzou This code change is the reason the number of violations increased in the pv smoothing test. The new AC look ahead number includes additional losses and therefore I'd argue it's more accurate. Let me know if it makes sense to update the numbers in the test or solve the test failure some other way.

Reply:
We should probably re-evaluate with the code from EPRI - although it will not include the additional losses - maybe if we use the updated pv profile as input to the Python from EPRI - we can match the violation count to the new SAM count...